### PR TITLE
Add clickbait

### PR DESCRIPTION
**Pull request by Tayyyyyyy**
:cl: Tayyyyyyy, bryanayalalugo
add: News reporters can now add titles to their stories!
/:cl:

Now that I've baited you into clicking on this PR...

This PR implements [this forum idea](https://nanotrasen.se/forum/topic/12193-news-header/).

You can add a title to your story:
newscaster1

The alert will display the title:
newscaster2

When viewed, it looks like this:
newscaster4

And it'll show up in the newspaper too:
newscaster3

I've also refactored a tiny bit and replaced trimming code with calls to `trim`.

### CODE REVIEW

File: code/game/machinery/newscaster.dm [hunk 1]
```diff
@@ -283,9 +288,7 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 		return 1
 
 	if(href_list["set_channel_name"])
-		channel_name = sanitizeSQL(strip_html_simple(input(usr, "Provide a Feed Channel Name", "Network Channel Handler", "")))
-		while(findtext(channel_name," ") == 1)
-			channel_name = copytext(channel_name, 2, lentext(channel_name) + 1)
+		channel_name = trim(sanitizeSQL(strip_html_simple(input(usr, "Provide a Feed Channel Name", "Network Channel Handler", ""))))
```
Comment (Tayyyyyyy) [inline code/game/machinery/newscaster.dm]: @tigercat2000 or @Fox-McCloud is `sanitizeSQL` here strictly necessary? It makes it so you can't use quotes in the feed channel, which is a bit annoying (I disabled sanitizeSQL when testing this PR). I don't think this is going into a SQL database, is it?

File: code/game/machinery/newscaster.dm [hunk 2]
```diff
@@ -42,8 +43,10 @@
 	is_admin_channel = 0
 	total_view_count = 0
 
-/datum/feed_channel/proc/announce_news()
-	return "Breaking news from [channel_name]!"
+/datum/feed_channel/proc/announce_news(title="")
+	if(title)
+		return "Breaking news from [channel_name]: [title]"
```
Comment (Citinited) [inline code/game/machinery/newscaster.dm]: There should probably be an exclamation mark here, for more !!excitement!!

Comment (Tayyyyyyy) [reply code/game/machinery/newscaster.dm]: Oh! Yeah, I took that exclamation off, cause an exclamation mark is already applied here:
https://github.com/ParadiseSS13/Paradise/blob/1877df65aa61016b928582d4e26d34310117cc22/code/game/machinery/newscaster.dm#L827

Comment (Citinited) [reply code/game/machinery/newscaster.dm]: oh, 👍 then

**Pull request merged**